addons/web/static/src/views/list/list_arch_parser.js

```diff
@@ -34,6 +34,31 @@ export class GroupListArchParser extends XMLParser {
     }
 }
 
+export class ControlListArchParser extends XMLParser {
+    parse(arch) {
+        const creates = [];
+        let buttonId = 0;
```

Comment:
ids won't be unique as we now have two "id generators" (one here, and one in the main parser)

Reply:
Maybe this parser is overkill, this code could be inlined in the main parser. It's probably better to only support `<create>` and `<button>` that are direct children anyway.